src/Common/tests/System/RealFormatterTestsBase.netcoreapp.cs

```diff
@@ -569,14 +1669,24 @@ private void TestFormatterDouble_Standard(double value, string format, string ex
         [InlineData(MathF.PI, "3.1415927")]
         [InlineData(0.0f, "0")]
         [InlineData(0.0046f, "0.0046")]
+        [InlineData(0.005f, "0.005")]
         [InlineData(0.125f, "0.125")]
-        [InlineData(0.845512390, "0.8455124")]
+        [InlineData(0.5f, "0.5")]
+        [InlineData(0.51f, "0.51")]
+        [InlineData(0.56789f, "0.56789")]
+        [InlineData(0.845512390f, "0.8455124")]
         [InlineData(1.0f, "1")]
-        [InlineData(429496.72, "429496.72")]
+        [InlineData(505.0f, "505")]
+        [InlineData(506.0f, "506")]
+        [InlineData(545.0f, "545")]
+        [InlineData(545.1f, "545.1")]
+        [InlineData(555.0f, "555")]
+        [InlineData(65747.125f, "65747.125")]
+        [InlineData(429496.72f, "429496.72")]
```

Comment:
This set of InlineDatas looks identical to the previous test's set.  Are they?  If so, can we put them into a shared MemberData?  Same for any other places where there's duplication.